batched-tx-disperse-multisend-py/src/constants.py

```diff
@@ -0,0 +1,5 @@
+"""Default values for the options / filters"""
```

Comment:
glad you are already thinking about filters. Looking at disperse app, we see that there are quite a lot of transactions, so I think we probably need a filter here. 

I recommend to put a filter based on value:
- min native distribution of 10ETH
- min ERC-20 distribution to 1% of token supply

Reply:
Exactly, this is where it's going!
So this is the right place to implement options?
Or is there a way to forward CLI flags for example?